src/args/__tests__/parse-helm-fetch-args.test.ts

```diff
@@ -0,0 +1,38 @@
+import { assertEquals } from "https://deno.land/std@0.86.0/testing/asserts.ts"
+import { parseHelmFetchArgs } from "../parse-helm-fetch-args.ts"
+
+Deno.test("Should parse helm fetch args", () => {
+  const args = parseHelmFetchArgs([
+    "upgrade",
+    "--install",
+    "one-service-release",
+    "tests/charts/one-service",
+    "--set",
+    "selector.app=my-app",
+    "-f",
+    "prod-values.yaml",
+    "--values",
+    "prod-values-2.yaml",
+    "--set-file",
+    "set-file-1.yaml",
+    "--set-string",
+    "my-string",
+    "--values=prod-values-3.yaml",
+    "--set=selector.app2=my-app-2",
+    "--set-file=set-file-2.yaml",
+    "--set-string=my-string",
+    "--repo",
+    "https://repo.com/helm/charts",
+    "--version",
+    "0.0.1",
+    "--deno-log-level",
+    "debug",
+  ])
+
+  assertEquals(args, [
+    "--repo",
+    "https://repo.com/helm/charts",
+    "--version",
+    "0.0.1",
```

Comment:
Надо больше флагов, вот такие ещё я выписал
```
--ca-file string             verify certificates of HTTPS-enabled servers using this CA bundle
--cert-file string           identify HTTPS client using this SSL certificate file
--devel                      use development versions, too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored.
--insecure-skip-tls-verify   skip tls certificate checks for the chart download
--key-file string            identify HTTPS client using this SSL key file
--keyring string             location of public keys used for verification (default "/Users/nitive/.gnupg/pubring.gpg")
--password string            chart repository password where to locate the requested chart
--repo string                chart repository url where to locate the requested chart
--username string            chart repository username where to locate the requested chart
--verify                     verify the package before using it
--version string             specify the exact chart version to use. If this is not specified, the latest version is used
--repository-cache string    path to the file containing cached repository indexes (default "/Users/nitive/Library/Caches/helm/repository")
--registry-config string     path to the registry config file (default "/Users/nitive/Library/Preferences/helm/registry.json")
```

Reply:
Two last options are global not only for fetch sub-command
```
--repository-cache string    path to the file containing cached repository indexes (default "/Users/nitive/Library/Caches/helm/repository")
--registry-config string     path to the registry config file (default "/Users/nitive/Library/Preferences/helm/registry.json")
```

Reply:
That's right but they must be passed to fetch command otherwise it won't work as expected

```sh
$ helm repo add my-repo https://charts.helm.sh/stable --registry-config ./config
"my-repo" has been added to your repositories

$ helm repo update
...Successfully got an update from the "my-repo" chart repository
Update Complete. ⎈Happy Helming!⎈

# Error here
$ helm deno template my-app my-repo/mysql --registry-config ./config
Could not find my-repo/mysql
Error: plugin "deno" exited with error

# No error without plugin
$ helm3 template my-app my-repo/mysql --registry-config ./config
---
# Source: mysql/templates/secrets.yaml
apiVersion: v1
kind: Secret
...
```